### PR TITLE
[ISSUE #3642] Display GID and Topic with namespace

### PR DESCRIPTION
**Pull request by zhaohai1299002788**
Specific GID and TID cannot be displayed for MQ instance with namespace

### CODE REVIEW

**Comment by zhaohai1299002788** [comment]
#3642

**Review by panzhi33 — APPROVED** [review]

**Review by guyinyou — APPROVED** [review]

File: tools/src/main/java/org/apache/rocketmq/tools/command/stats/StatsAllSubCommand.java [hunk 1]
```diff
@@ -94,22 +94,22 @@ public static void printTopicDetail(final DefaultMQAdminExt admin, final String
...
-                    System.out.printf("%-32s  %-32s %12d %11.2f %11.2f %14d %14d%n",
-                        UtilAll.frontStringAtLeast(topic, 32),
-                        UtilAll.frontStringAtLeast(group, 32),
-                        accumulate,
-                        inTPS,
-                        outTPS,
-                        inMsgCntToday,
-                        outMsgCntToday
+                    System.out.printf("%-64s  %-64s %12d %11.2f %11.2f %14d %14d%n",
+                            UtilAll.frontStringAtLeast(topic, 64),
```
Comment (HScarb) [inline tools/src/main/java/org/apache/rocketmq/tools/command/stats/StatsAllSubCommand.java:98]: the indent of code does not match `rmq_codeStyle.xml`
see https://rocketmq.apache.org/docs/code-guidelines/

Comment (zhaohai1299002788) [reply tools/src/main/java/org/apache/rocketmq/tools/command/stats/StatsAllSubCommand.java]: fix code indentation

**Comment by MatrixHB** [comment]
I think it is feasible to increase the string display length, but I have another method to solve this problem. Using NamespaceUtil.withoutNamespace to hide the MQ instance ID
```
System.out.printf("%-32s  %-32s %12d %11.2f %11.2f %14d %14d%n",
                        UtilAll.frontStringAtLeast(NamespaceUtil.withoutNamespace(topic), 32),
                        UtilAll.frontStringAtLeast(NamespaceUtil.withoutNamespace(group), 32),
                        accumulate,
                        inTPS,
                        outTPS,
                        inMsgCntToday,
                        outMsgCntToday
                    );
```

**Comment by panzhi33** [comment]
> I think it is feasible to increase the string display length, but I have another method to solve this problem. Using NamespaceUtil.withoutNamespace to hide the MQ instance ID
> 
> ```
> System.out.printf("%-32s  %-32s %12d %11.2f %11.2f %14d %14d%n",
>                         UtilAll.frontStringAtLeast(NamespaceUtil.withoutNamespace(topic), 32),
>                         UtilAll.frontStringAtLeast(NamespaceUtil.withoutNamespace(group), 32),
>                         accumulate,
>                         inTPS,
>                         outTPS,
>                         inMsgCntToday,
>                         outMsgCntToday
>                     );
> ```

This will cause problems with duplicate names

**Comment by zhaohai1299002788** [comment]
> 我认为增加字符串显示长度是可行的，但我有另一种方法来解决这个问题。使用 NamespaceUtil.withoutNamespace 隐藏 MQ 实例 ID
> 
> ```
> System.out.printf("%-32s  %-32s %12d %11.2f %11.2f %14d %14d%n",
>                         UtilAll.frontStringAtLeast(NamespaceUtil.withoutNamespace(topic), 32),
>                         UtilAll.frontStringAtLeast(NamespaceUtil.withoutNamespace(group), 32),
>                         accumulate,
>                         inTPS,
>                         outTPS,
>                         inMsgCntToday,
>                         outMsgCntToday
>                     );
> ```
Topics and GIDs with namespaces cannot be hidden, they need to be displayed。

**Comment by MatrixHB** [comment]
> > 我认为增加字符串显示长度是可行的，但我有另一种方法来解决这个问题。使用 NamespaceUtil.withoutNamespace 隐藏 MQ 实例 ID
> > ```
> > System.out.printf("%-32s  %-32s %12d %11.2f %11.2f %14d %14d%n",
> >                         UtilAll.frontStringAtLeast(NamespaceUtil.withoutNamespace(topic), 32),
> >                         UtilAll.frontStringAtLeast(NamespaceUtil.withoutNamespace(group), 32),
> >                         accumulate,
> >                         inTPS,
> >                         outTPS,
> >                         inMsgCntToday,
> >                         outMsgCntToday
> >                     );
> > ```
> 
> Topics and GIDs with namespaces cannot be hidden, they need to be displayed。The same name may be notified at the same time。

OK, I get it.

**Comment by vongosling** [comment]
Could you help to follow our community pr rule, just like we have done for your other pr?

**Review by odbozhou — APPROVED** [review]

**Pull request merged**